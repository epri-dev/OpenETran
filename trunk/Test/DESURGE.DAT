1 3 100.0 1 1 0.01e-6 100.0e-6

cable 1 2.0 1.5e8 0.0e3

surge 100.0e3 1.2e-6 50.0e-6 0.0e-6
pairs 1 0
poles 2

insulator 100.0e3 0.0 5.42434 8.4265e21
pairs 1 0
poles 2

meter
pairs 1 0
poles 2


