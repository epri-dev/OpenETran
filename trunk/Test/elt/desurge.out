  N   span     dT   Tmax
  1 100.00 1e-008 0.0001
Z-phase
  2.00000e+000
Z-modal
  2.00000e+000
Modal Transformation (Ti)
  1.00000e+000
Power-frequency source at pole 1.
   #        vp        vm         i
   1       0.0       0.0       0.0
Power-frequency source at pole 3.
   #        vp        vm         i
   1       0.0       0.0       0.0
Meter at pole 2, from 1 to 0, max V   = 9.999931e+004
Insulator at pole 2, from 1 to 0 flashed over at 9.710000e-006 seconds
nr_iter = 0, nr_max = 0
Execution time [sec]: 0.078
